Editor/AGS.Editor/Panes/ScriptEditor.cs

```diff
@@ -344,7 +344,7 @@ private void AutoComplete_BackgroundCacheUpdateStatusChanged(BackgroundAutoCompl
             {
                 if (this.IsHandleCreated)
                 {
-                    this.Invoke(new AnonymousDelegate(UpdateFunctionList));
+                    this.Invoke(new Action(_anonymousDelegate));

```

Comment:
This looks like a mistake; you are replacing a invoke to UpdateFunctionList with anonymousDelegate.